bindings/go/test.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	sdk "example.com/bindings/iota_sdk_ffi"
+)
+
+func main() {
+	client := sdk.GraphQlClientNewDevnet()
+
+	chainID, err := client.ChainId()
+	// if err != nil {
+	// 	log.Fatalf("Failed to get chain ID: %#v (%T)", err, err)
+	// }
```

Comment:
why not keep it uncommented?

Reply:
Because it's actually not nil and I have no clue how to properly test it

Reply:
Then remove it completely or do you think it might be useful to investigate further in the future?

Reply:
I would rather actually understand what's wrong and fix it tbh